src/control/src/point_solver/psolver.cpp

```diff
@@ -64,49 +63,88 @@ std::pair<Position, int> PointSolver::update_closest_point(
  * @return std::pair<Position, int> lookahead point and error status (1 = error)
```

Comment:
This return type is not up to date. Update the documentation. If you want, you may erase this one and correct the hpp only